src/configurations/destinations/bq/ui-config.json

```diff
@@ -226,6 +226,17 @@
           "footerNote": "Enable this feature to skip sending the event data to the “tracks” table",
           "default": false
         },
+        {
+          "type": "checkbox",
+          "label": "Skip Views Creation",
+          "value": "skipViews",
+          "default": false,
+          "immutable": true,
+          "footerURL": {
+            "link": "https://www.rudderstack.com/docs/destinations/warehouse-destinations/bigquery/#partitioned-tables-and-views",
+            "text": "Enable this feature to skip creation of views in BigQuery. Learn more about views"
```

Comment:
Enable this feature -> Enable this checkbox ?

Reply:
Making it `Enable this toggle`, [checkbox would mean different](https://www.google.com/search?q=checkbox+vs+toggle&sca_esv=9c61c6d612e03081&rlz=1C5GCEM_en&udm=2&biw=1920&bih=992&sxsrf=AHTn8zrbSmtLQHdy0t90sXv7OYUp5v_ynA%3A1742465874717&ei=UuvbZ_StK6aIg8UPi9vSkAM&ved=0ahUKEwj0nvubt5iMAxUmxKACHYutFDIQ4dUDCBQ&uact=5&oq=checkbox+vs+toggle&gs_lp=EgNpbWciEmNoZWNrYm94IHZzIHRvZ2dsZTIFEAAYgAQyBhAAGAUYHjIGEAAYCBgeMgYQABgIGB5IpRBQzgRYjw9wAXgAkAEAmAFtoAGdB6oBAzkuMbgBA8gBAPgBAZgCC6ACuAfCAgcQIxgnGMkCwgIGEAAYBxgewgIKEAAYgAQYQxiKBcICCBAAGIAEGLEDwgINEAAYgAQYsQMYQxiKBcICBxAAGIAEGArCAggQABgIGAoYHpgDAIgGAZIHAzkuMqAHjyeyBwM4LjK4B7UH&sclient=img#vhid=rXRucSwCTBeiEM&vssid=mosaic).